typify-impl/tests/vega.out

```diff
@@ -6287,21 +6985,21 @@ pub struct EverythingSubtype1 {
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub width: Option<NumberOrSignal>,
 }
-#[derive(Clone, Debug, Deserialize, Serialize)]
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
 pub struct Expr {
     #[serde(rename = "as", default, skip_serializing_if = "Option::is_none")]
     pub as_: Option<String>,
     pub expr: String,
 }
-#[derive(Clone, Debug, Deserialize, Serialize)]
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
 pub struct ExprString(pub String);
 impl std::ops::Deref for ExprString {
     type Target = String;
     fn deref(&self) -> &String {
         &self.0
     }
 }
-#[derive(Clone, Debug, Deserialize, Serialize)]
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
 #[serde(deny_unknown_fields)]
 pub struct ExtentTransform {
```

Comment:
Just for example:, but this type clearly doesn't have a default value in the json spec.